spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -735,31 +735,43 @@ def get_place_sensor(self):
         # the robot is at the place receptacle
         gripper_T_base = self.get_in_gripper_tf()
         base_T_gripper = gripper_T_base.inverted()
-        base_frame_place_target = self.get_base_frame_place_target()
+        base_frame_place_target = self.get_base_frame_place_target_spot()
         hab_place_target = self.spot2habitat_translation(base_frame_place_target)
         gripper_pos = base_T_gripper.transform_point(hab_place_target)
 
         return gripper_pos
 
-    def get_base_frame_place_target(self):
+    def get_base_frame_place_target_spot(self):
         if self.place_target_is_local:
             base_frame_place_target = self.place_target
         else:
             base_frame_place_target = self.get_target_in_base_frame(self.place_target)
         return base_frame_place_target
 
-    def get_place_distance(self):
-        gripper_T_base = self.get_in_gripper_tf()
-        base_frame_gripper_pos = np.array(gripper_T_base.translation)
-        base_frame_place_target = self.get_base_frame_place_target()
-        hab_place_target = self.spot2habitat_translation(base_frame_place_target)
-        hab_place_target = np.array(hab_place_target)
-        place_dist = np.linalg.norm(hab_place_target - base_frame_gripper_pos)
-        xy_dist = np.linalg.norm(
-            hab_place_target[[0, 2]] - base_frame_gripper_pos[[0, 2]]
+    def get_base_frame_place_target_hab(self):
+        """
+        Returns the current place target in the base frame as (x,z,-y) as per Habitat convention
+        """
+        base_frame_place_target_spot = self.get_base_frame_place_target_spot()
+        base_frame_place_target_hab = np.array(
+            self.spot2habitat_translation(base_frame_place_target_spot)
         )
-        z_dist = abs(hab_place_target[1] - base_frame_gripper_pos[1])
-        return place_dist, xy_dist, z_dist
+        return base_frame_place_target_hab
```

Comment:
There are 2 conventions for reference frames in `spot-sim2real` - spot & habitat.

This adds getters for place_target in base frame for both these conventions